pytorch_lightning/accelerators/accelerator.py

```diff
@@ -93,7 +93,7 @@ def backward(self, closure_loss, optimizer, opt_idx, *args, **kwargs):
             )
         else:
             # do backward pass
-            model = self.trainer.get_model()
+            model = self.trainer.model
```

Comment:
ummm... this will break the other accelerators no?
DP will be wrapped, and so will the ddp one?

Reply:
Right, this is motivated by DDP. `self.trainer.model` would be an instance of `DistributedDataParallel`. We want to call backwards on that as opposed to th emodule inside. Motivating factor: gradient accumulation in #4301. While accumulating gradients, if we call backwards using the module inside DDP, we ignore the `require_backward_grad_sync=False` set on DDP by `no_sync` meaning we still run the backwards pass. am i making sense?

Reply:
Should we not be doing the backward pass on the wrapped model? Why would we only call `backwards()` on the internal module?